packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-corporation/template.html

```diff
@@ -0,0 +1,96 @@
+<p
+    class="oui-heading_3 mb-1 mt-3"
+    data-translate="user_account_identity_documents_corporation_requirements"
+></p>
+<p
+    data-translate="user_account_identity_documents_corporation_requirements_intro"
+></p>
```

Comment:
I think this could be kept inside the user_identity-documents component as I think it will be the same for all cases

Reply:
That's not really the same text between standard text and corporation/individual text. We could probably replace the standard text by the new one but I don't know if it's expected.